src/pudl/metadata/resources/ferc1.py

```diff
@@ -212,13 +212,13 @@
             "fields": [
                 "utility_id_ferc1",
                 "report_year",
+                "depreciation_type",
                 "plant_function",
                 "plant_status",
                 "utility_type",
                 "ending_balance",
                 "record_id",
                 "balance",
-                "ferc_account",
```

Comment:
The integration tests are failing with an error about `ferc_account` being missing in `enforce_schema()`

Reply:
Ah, these column changes needed to be made in the denormalized version of the table too.